jwst/extract_1d/extract.py

```diff
@@ -2656,9 +2656,11 @@ def run_extract1d(
     """
     # Read and interpret the extract1d reference file.
     try:
-        ref_dict = open_extract1d_ref(extract_ref_name, input_model.meta.exposure.type)
+        ref_dict = open_extract1d_ref(extract_ref_name, input_model.meta.exposure.type,
+                                      input_model.meta.model_type)
     except AttributeError:  # Input is a ModelContainer of some type
-        ref_dict = open_extract1d_ref(extract_ref_name, input_model[0].meta.exposure.type)
+        ref_dict = open_extract1d_ref(extract_ref_name, input_model[0].meta.exposure.type,
```

Comment:
Should've quoted this too, for the same reason.

Reply:
@tapastro I think this is ready to be merged